cranelift/simplejit/src/backend.rs

```diff
@@ -661,26 +697,16 @@ fn lookup_with_dlsym(name: &str) -> *const u8 {
             if addr.is_null() {
                 continue;
             }
-            return addr as *const u8;
+            return Some(addr as *const u8);
         }
 
-        let msg = if handles[1].is_null() {
-            "(msvcrt not loaded)"
-        } else {
-            ""
-        };
-        panic!("cannot resolve address of symbol {} {}", name, msg);
+        None
     }
 }
 
+#[derive(Default)]
 struct SimpleJITRelocSink {
-    pub relocs: Vec<RelocRecord>,
-}
-
-impl SimpleJITRelocSink {
-    pub fn new() -> Self {
-        Self { relocs: Vec::new() }
-    }
+    relocs: Vec<RelocRecord>,
```

Comment:
`relocs` should be `pub` for this to be useful to an end user

Reply:
This type isn't public. It is only used to collect relocations for future usage by simplejit.